virtualizarr/readers/dmrpp.py

```diff
@@ -0,0 +1,331 @@
+from typing import Optional
+from xml.etree import ElementTree as ET
```

Comment:
Is this the only extra import required? (And this is a built-in python library module right?)

Reply:
Yes this is the only extra import and it is built in